examples/subclass/plot_printgraph.py

```diff
@@ -8,18 +8,17 @@
 
 import matplotlib.pyplot as plt
 import networkx as nx
-from networkx import Graph
```

Comment:
In this instance it might actually be more clear to do this and use `Graph` instead of `nx.Graph` in the subsequent subclass definition. I don't think it's common to keep the parent namespace when defining a custom subclass.

Reply:
This might be a chance for me to learn more about the class namespaces. When you say "keep the parent namespace" do you mean that the `nx` namespace gets included in the new class's namespace somehow? Or are you referring to _using_ the parent namespace in the code for the subclass? Or maybe something else? :}

Reply:
Maybe this is a place for `super` instead of `nx.Graph.`?

Reply:
> This might be a chance for me to learn more about the class namespaces.

Not unless I improve the clarity of my wording!

> When you say "keep the parent namespace" do you mean that the nx namespace gets included in the new class's namespace somehow?

No, my comment was really only about style: I feel like I see the

```python
from parentmod import ABaseClass

class MyClass(ABaseClass):
...
```
pattern more often than 

```python
import parentmod as pm

class MyClass(pm.ABaseClass):
...
```

This is entirely anecdotal on my part!

> Maybe this is a place for super instead of nx.Graph.? I'll bet this was written before super...

+1

Reply:
See  a444bc039.